content/docs/2_cookbook/7_extensions/0_first-panel-field/cookbook-recipe.txt

```diff
@@ -149,7 +149,7 @@ In the single file component, we recreate the template inside `template` tags:
 
 We are also adding the script tags that will shortly contain the logic for the field and an optional style tag (we will not use custom styles for this example, so you might as well omit them).
 
-In `/src/index.php`, we import this new component, assign the component to the `doi` field and remove the `template` property from the first iteration, so that it looks like this:
+In `/src/index.js`, we import this new component, assign the component to the `doi` field and remove the `template` property from the first iteration, so that it looks like this:
 
 ```js "site/plugins/doifield/src/components/fields/DoiField.vue"
```

Comment:
Shouldn't this also be:

```suggestion
```js "site/plugins/doifield/src/index.js"
```

Reply:
Was just fixing this...

Reply:
You are too fast for me ;)